### ISSUE TITLE
Datagrid tests broken by BYTES_LENGTH8 handling change

### ISSUE THREAD

**Issue by alamar** [OpenHFT/Chronicle-Wire#389]
After fa487adbb9f3517185e7fcf744b89de93376ddd9, 9 tests in Datagrid fail:
https://teamcity.chronicle.software/buildConfiguration/Chronicle_BuildAll_EnterpriseJava11/652508

The failure may be pinpointed to a single line change:
```
diff --git a/src/main/java/net/openhft/chronicle/wire/BinaryWire.java b/src/main/java/net/openhft/chronicle/wire/BinaryWire.java
index 4dff5f0f7..aedce9fad 100644
--- a/src/main/java/net/openhft/chronicle/wire/BinaryWire.java
+++ b/src/main/java/net/openhft/chronicle/wire/BinaryWire.java
@@ -3679,7 +3679,7 @@ public class BinaryWire extends AbstractWire implements Wire {
             switch (code >> 4) {
                 case BinaryWireHighCode.CONTROL:
                     switch (code) {
-                        case BYTES_LENGTH8:
+//                      case BYTES_LENGTH8:
                         case BYTES_LENGTH16:
                         case BYTES_LENGTH32: {
                             if (using instanceof StringBuilder) {
```

**Comment by peter-lawrey** [comment]
thank you.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.22.21](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.22.21), [BOM-2.22.150](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.22.150)
